docs/send-data/opentelemetry-collector/remote-management/source-templates/apache.md

```diff
@@ -0,0 +1,102 @@
+---
+id: apache
+title: Apache Source Template
+sidebar_label: Apache
+description: Learn about the Sumo Logic Apache source template for OpenTelemetry.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+<p><a href="/docs/beta"><span className="beta">Beta</span></a></p>
+
+<img src={useBaseUrl('img/integrations/web-servers/apache.png')} alt="Thumbnail icon" width="100"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="30"/>
+
+Apache source template creates an OpenTelemetry configuration that can be pushed to a remotely managed OpenTelemetry collector. By creating this source template and pushing the config to the appropriate OpenTelemetry agent you can ensure collection of logs and metrics of Apache to Sumo Logic.
```

Comment:
Adding this definition here because it's used later in the doc

```suggestion
Apache source template creates an OpenTelemetry configuration that can be pushed to a remotely managed OpenTelemetry collector (abbreviated as otelcol). By creating this source template and pushing the config to the appropriate OpenTelemetry agent you can ensure collection of logs and metrics of Apache to Sumo Logic.
```